kbase_cache_client/kbase_cache_client.py

```diff
@@ -3,46 +3,38 @@
 import configparser
 import os
 from pprint import pprint as pp
-from .exceptions import NoCacheIdentifiers, HTTPRequestError, UnknownRequestError, DownloadDirNotWriteable, \
-    CacheNonexistent, AuthorizationTokenNotSet
+from .exceptions import (NoCacheIdentifiers, HTTPRequestError,
+                         UnknownRequestError, DownloadDirNotWriteable,
+                         CacheNonexistent, AuthorizationTokenNotSet)
 
 config = configparser.ConfigParser()
 if os.path.exists('test.cfg'):
     config.read('test.cfg')
-
     if not config.get('KBASE_CACHE_SERVICE', 'TOKEN', fallback=None):
         if not os.getenv('KBASE_CACHE_TOKEN', None):
             raise IOError('Please set your service token in test.cfg, as:\n'
                           '[KBASE_CACHE_SERVICE]\nTOKEN=<token>.\n'
                           'Or as an environmental variable KBASE_CACHE_TOKEN\n'
                           'Consult KBase Administrators if you are not sure how to generate a token')
-else:
```

Comment:
I want to be able to pass the token as a parameter when creating the client, so I'd rather the env var is optional.